src/debugger/appWorker.ts

```diff
@@ -60,9 +60,9 @@ export class SandboxedAppWorker {
 
     private static PROCESS_MESSAGE_INSIDE_SANDBOX = "onmessage({ data: postMessageArgument });";
 
-    constructor(sourcesStoragePath: string, debugAdapterPort: number, postReplyToApp: (message: any) => void, {
+    constructor(private packagerPort: number, sourcesStoragePath: string, debugAdapterPort: number, postReplyToApp: (message: any) => void, {
```

Comment:
I'm fine with the usage of this feature for simple/trivial constructors:

```
class Person {
    constructor(public firstName: string, public lastName: string) {}
}
```

But as soon as we start having logic inside the constructor, I'd prefer us to be explicit. Also I'd strongly argue we should not mix the two.

```
class Foo {
    constructor(bar: Bar, name: string) {
        this.name = name;

        bar.init(this.name);
        // More logic follows...
    }
}
```